src/System.Numerics.Tensors/ref/System.Numerics.Tensors.cs

```diff
@@ -0,0 +1,151 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+using System.Collections;
+using System.Collections.Generic;
+
+namespace System.Numerics.Tensors
+{
+    public static class ArrayTensorExtensions
```

Comment:
@ericstj - what do you think about removing these extensions for now? Or do you feel they should be kept in for `0.1`?

Reply:
They aren't hurting anything and we'll want to support the scenario.  Did you have something in mind about them.

One thing that comes to mind is the whole 'reverseStride' terminology.  I wish we could come up with a better way to phrase that.

Reply:
> Did you have something in mind about them.

I just don't see how they are necessary/critical for this first release. What's the scenario driving them? I know we can remove them later if they are unnecessary, but I was thinking we'd take the opposite approach and only put in what we know is needed now.

> One thing that comes to mind is the whole 'reverseStride' terminology. I wish we could come up with a better way to phrase that.

My vote is still for `rowMajor` and `columnMajor`. I know that `row` and `column` don't make sense when talking about n-dimensions, but it is a lot easier to explain to people.

https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.html

> order : {‘C’, ‘F’}, optional
Row-major (C-style) or column-major (Fortran-style) order.